dl/AlexNet/requirements.txt

```diff
@@ -0,0 +1,25 @@
+-i https://pypi.org/simple
```

Comment:
There are several libraries listed that were not used in the codebase. Please remove them.

Reply:
made necessary changed to requirements.txt